layer1/layer1.yaml

```diff
@@ -39,7 +39,7 @@ components:
             - speed_100_gbps
             - speed_200_gbps
             - speed_400_gbps
-          default: speed_10_gbps
+          default: speed_1_gbps
```

Comment:
Disagree as it was changed to be 10 from 1.

Reply:
Changed